### PR TITLE
Implement CC/BCC

### PR DESCRIPTION
**Pull request by Goorzhel**
Well, I tried.

I wasn't sure how best to pass the CC/BCCs (the lack of DRY was killing me) but I couldn't think of anything better — rather, more readable — than two variables. (I thought of using a list of strings but that'd just be goofy.)

Anyway, tested it through my own account, looks workable to me.

### CODE REVIEW

File: gmailer_oauth.py [hunk 1]
```diff
@@ -18,14 +18,14 @@
 APP_NAME = 'gmailer-oauth'
 
 
-def prepare_message(to, subject, message_text, attachment):
+def prepare_message(to, subject, message_text, attachment,cc=None,bcc=None):
```
Comment (ammgws) [inline gmailer_oauth.py]: add space after comma

File: gmailer_oauth.py [hunk 2]
```diff
@@ -34,6 +34,10 @@ def create_message_with_attachment(to, subject, message_text, attachment):
     message = MIMEMultipart('related')
     message['to'] = to
     message['subject'] = subject
+    if cc:
```
Comment (ammgws) [inline gmailer_oauth.py]: Since `cc` and `bcc` default to `None` we can remove the checks here and just have `message['cc'] = cc`

File: gmailer_oauth.py [hunk 3]
```diff
@@ -34,6 +34,10 @@ def create_message_with_attachment(to, subject, message_text, attachment):
...
+        message['cc'] = cc
+    if bcc:
```
Comment (ammgws) [inline gmailer_oauth.py]: Same as above

File: gmailer_oauth.py [hunk 4]
```diff
@@ -58,11 +62,16 @@ def create_message_with_attachment(to, subject, message_text, attachment):
     return message.as_bytes()
 
 
-def create_message(to, subject, message_text):
+def create_message(to, subject, message_text, cc, bcc):
     """Returns a RFC2822 formatted email message as a base64url encoded string."""
     message = MIMEText(message_text)
     message['to'] = to
     message['subject'] = subject
+    if cc:
```
Comment (ammgws) [inline gmailer_oauth.py]: Same as above

File: gmailer_oauth.py [hunk 5]
```diff
@@ -58,11 +62,16 @@ def create_message_with_attachment(to, subject, message_text, attachment):
...
+        message['cc'] = cc
+    if bcc:
```
Comment (ammgws) [inline gmailer_oauth.py]: Same as above

File: gmailer_oauth.py [hunk 6]
```diff
@@ -116,7 +135,7 @@ def create_dir(ctx, param, directory):
 )
 @click.option('--dry-run', is_flag=True)
 @click.option('--interactive', '-i', is_flag=True)
-def main(config_path, cache_path, recipient, message, subject, dry_run, client_id, client_secret, attachment=None):
+def main(config_path, cache_path, recipient, message, subject, dry_run, interactive, client_id, client_secret, cc=None, bcc=None, attachment=None):
```
Comment (ammgws) [inline gmailer_oauth.py]: Let's add interactive in another PR (this one should just be for CC/BCC)

File: gmailer_oauth.py [hunk 7]
```diff
@@ -129,11 +148,12 @@ def main(config_path, cache_path, recipient, message, subject, dry_run, client_i
         os.makedirs(cache_dir, exist_ok=True)
 
     config_file = os.path.join(config_path, 'config.ini')
-    logging.debug('Using config file: %s', config_file)
-    config = ConfigParser()
-    config.read(config_file)
-    client_id = config.get('Gmail', 'client_id')
-    client_secret = config.get('Gmail', 'client_secret')
+    if os.path.isfile(config_file):
```
Comment (ammgws) [inline gmailer_oauth.py]: Let's do this in another PR (this one whould just be for CC/BCC)

**Review by ammgws — CHANGES_REQUESTED** [review]
Good work. Tested and worked here (though we should really add some proper tests...).

Just have a couple requests below. I was going to just do them myself but thought you might like to do it.

**Review by ammgws — APPROVED** [review]

**Comment by Goorzhel** [comment]
Yeah, I've never written unit tests either (imagine that), but I'm down to learn.

All requested changes fixed. I'll move the bugfixes to another branch.
